README.md

```diff
@@ -57,6 +60,53 @@ Note that these locks are only enforced when using `#with_lock`, not at the data
 
 More in-depth method documentation can be found at [rdoc.info](http://rdoc.info/github/mongoid/mongoid-locker/frames).
 
+### Customizable :locked_at and :locked_until field names
+By default, Locker uses fields with `:locked_at` and `:locked_until` names which should be defined in a model.
+```ruby
+class User
+  include Mongoid::Document
+  include Mongoid::Locker
+
+  field :locked_at, type: Time
+  field :locked_until, type: Time
+end
+```
+
+Use `Mongoid::Locker.configure` to setup field names which used by Locker for all models where it's included.
+```ruby
+Mongoid::Locker.configure do |config|
+  config.locked_at_field = :global_locked_at
+  config.locked_until_field = :global_locked_until
+end
+
+class User
+  include Mongoid::Document
+  include Mongoid::Locker
+
+  field :global_locked_at, type: Time
+  field :global_locked_until, type: Time
+end
+```
+
+`locker` method in your model accepts `:locked_at_field` and `:locked_until_field` options to setup field names which used by Locker for the model.
```

Comment:
The `locker` method ..

maybe add

_This can be useful when another popular library uses the same field for different purposes._